### PR TITLE
RHDM-1788: changes related to upgrade of sshd-core 2.9.2

### PR DESCRIPTION
**Pull request by mareknovotny**
Upgrading sshd-core to 2.9.2

https://issues.redhat.com/browse/RHDM-1788 and https://issues.redhat.com/browse/RHPAM-3807

** Parent and dependent PRs **: 
* https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/2202
* https://github.com/kiegroup/kie-wb-common/pull/3801
* https://github.com/kiegroup/jbpm-wb/pull/1587

<details>
<summary>
How to replicate CI configuration locally?
</summary>

Build Chain tool does "simple" maven build(s), the builds are just Maven commands, but because the repositories relates and depends on each other and any change in API or class method could affect several of those repositories there is a need to use [build-chain tool](https://github.com/kiegroup/github-action-build-chain) to handle cross repository builds and be sure that we always use latest version of the code for each repository.
 
[build-chain tool](https://github.com/kiegroup/github-action-build-chain) is a build tool which can be used on command line locally or in Github Actions workflow(s), in case you need to change multiple repositories and send multiple dependent pull requests related with a change you can easily reproduce the same build by executing it on Github hosted environment or locally in your development environment. See [local execution](https://github.com/kiegroup/github-action-build-chain#local-execution) details to get more information about it.
</details>

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
 
* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>
  
* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

**Comment by mareknovotny** [comment]
closed in favor of https://github.com/kiegroup/appformer/pull/1374

File: uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/main/java/org/uberfire/java/nio/fs/jgit/daemon/ssh/GitSSHService.java
Comment (mareknovotny) [inline uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/main/java/org/uberfire/java/nio/fs/jgit/daemon/ssh/GitSSHService.java:66]: removed managed Ciphers as we discussed in our chat

**Review by RishiRajAnand — APPROVED** [review]

**Comment by mareknovotny** [comment]
jenkins do cdb

**Review by domhanak — APPROVED** [review]
Well LGTM if we have green builds. Once merged I can double check nightlys and if we have issues we can react.

**Comment by mareknovotny** [comment]
jenkins do fdb

**Comment by mareknovotny** [comment]
jenkins do cdb

**Comment by mareknovotny** [comment]
jenkins do fdb

**Comment by mareknovotny** [comment]
jenkins do fdb

**Comment by mareknovotny** [comment]
tests now in FDB fails due old Firefox ESR installed on build node

**Comment by mareknovotny** [comment]
jenkins do fdb

**Comment by akumar074** [comment]
Jenkins run fdb

**Comment by mareknovotny** [comment]
jenkins do cdb

**Comment by mareknovotny** [comment]
jenkins do fdb

**Comment by mareknovotny** [comment]
jenkins do fdb

**Comment by mareknovotny** [comment]
FDB on https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/2202 is green so all is ready

**Pull request merged**
